patterns/project-layout.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Title: Project Layout
+ * Slug: twentytwentyfour/project-layout
+ * Categories: text
+ */
+
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"6vw","right":"6vw","bottom":"6vw","left":"6vw"}},"elements":{"link":{"color":{"text":"#fffffffc"}}},"color":{"background":"#181818","text":"#fffffffc"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-text-color has-background has-link-color" style="color:#fffffffc;background-color:#181818;padding-top:6vw;padding-right:6vw;padding-bottom:6vw;padding-left:6vw"><!-- wp:columns {"verticalAlignment":null,"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|50","left":"var:preset|spacing|50"}}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"verticalAlignment":"stretch","width":"50%","style":{"spacing":{"padding":{"right":"0%"}}}} -->
+<div class="wp-block-column is-vertically-aligned-stretch" style="padding-right:0%;flex-basis:50%"><!-- wp:group {"style":{"dimensions":{"minHeight":"100%"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between","justifyContent":"stretch"}} -->
+<div class="wp-block-group" style="min-height:100%"><!-- wp:image {"id":106,"sizeSlug":"large","linkDestination":"none"} -->
+<figure class="wp-block-image size-large"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/tt4_portfolio_9-scaled-1.jpg " alt="" class="wp-image-106"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontSize":"0.8rem"}}} -->
+<p style="font-size:0.8rem"><?php echo esc_html__( "1. Queste basi mi hanno permesso di affrontare la sessione con un profondo apprezzamento per l'identità dell'hotel e una chiara visione della storia che volevo raccontare attraverso il mio obiettivo.", 'twentytwentyfour' ); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"33%"} -->
+<div class="wp-block-column" style="flex-basis:33%"><!-- wp:group {"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between","justifyContent":"stretch"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"style":{"typography":{"lineHeight":"1.2","fontSize":"1.4rem"}}} -->
+<p style="font-size:1.4rem;line-height:1.2"><strong><?php echo esc_html__( "La mia missione era tradurre la miscela unica di arte, artigianato e bellezza naturale dell'hotel in una raccolta di fotografie evocative che risuonassero con gli spettatori e trasmettessero lo spirito di questa incantevole destinazione. Mi sono avvicinato all'incarico con l'obiettivo di catturare il carattere dell'hotel nella sua interezza. Ogni fotografia è diventata una testimonianza della fusione tra arte e natura, celebrando l'impegno dell'hotel per l'autenticità e la sua armoniosa convivenza con il paesaggio circostante.", 'twentytwentyfour' ); ?></strong></p>
```

Comment:
This text shouldn't be bold

Reply:
I used to the new screen into Figma, where u can see the new edit: https://www.figma.com/file/AlYr03vh4dVimwYwQkTdf6/Twenty-Twenty-Four?type=design&node-id=119-543&mode=design&t=ULaidtXPOPeoOe6Z-0